CHANGES.rst

```diff
@@ -40,6 +40,7 @@ Bugfixes
 - Fix event header bar in IE11 (:issue:`3135`)
 - Fix footer on login page (:issue:`3132`)
 - Use correct module name for abstract notification emails in the event log
+- Fix iterdays function and add unit tests for it.
```

Comment:
That's too lowlevel/technical for the changelog.

    - Do not show an extra day in timetable management if an event
      begins before a DST change